frontend/catalyst/jax_tracer.py

```diff
@@ -201,7 +200,7 @@ def _eval_jaxpr(*args, **kwargs):
 # Take care when adding primitives to this set in order to avoid introducing a quadratic number of
 # edges to the jaxpr equation graph in ``sort_eqns()``. Each equation with a primitive in this set
 # is constrained to occur before all subsequent equations in the quantum operations trace.
-FORCED_ORDER_PRIMITIVES = {qdevice_p, gphase_p}
+FORCED_ORDER_PRIMITIVES = {qdevice_p}
```

Comment:
TODO: This needs to be handled in another way.

Reply:
Oh I wonder if this is the reason the primitive exists. Without qubits, it could be reordered before the device is instantiated, which would result in a crash I think